crates/polars-parquet/src/arrow/read/deserialize/nested.rs

```diff
@@ -202,6 +202,17 @@ pub fn columns_to_iter_recursive(
             )?
             .collect_n(filter)?
         },
+        Binary | Utf8 => {
```

Comment:
We don't support those types. I think we directly use `BinviewDecoder` here if we can.

Reply:
We will leave this for this PR. But we should be able to remove the code that generates `(Large)Utf8`, `(LargeBinary)` in favor of `BinaryView`.